### DISCUSSION TITLE
algebra relazionale - 3

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#60] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/2158e1d1-df30-47bb-8fec-303145d4eaca)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/a8ace248-0696-451e-bf85-2a78dd9cfa4b)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/79638284/3a5002dd-cfb4-44d3-9a2e-8ca7cbbb67e0)
